### PR TITLE
Fix issue with getting the latest Hugo release

### PR DESCRIPTION
**Pull request by IngridMorstrad**
Fix is to change to -7 as documented in the issue

https://github.com/victoriadrake/hugo-latest-cd/issues/1

### CODE REVIEW

File: action.sh [hunk 1]
```diff
@@ -7,9 +7,9 @@ echo '🔧 Install tools'
 npm init -y && npm install -y postcss-cli autoprefixer
 
 echo '🤵 Install Hugo'
-HUGO_VERSION=$(curl -s https://api.github.com/repos/gohugoio/hugo/releases/latest | jq -r '.tag_name')
+HUGO_VERSION='v0.109.0'
```
Comment (victoriadrake) [inline action.sh]: Why is this version pinned here?

File: hugo-latest-cd.yml [hunk 2]
```diff
@@ -12,22 +12,23 @@ jobs:
...
-          node-version: 12.x
+          node-version: 14.x
       - name: 🔧 Install tools
         run: |
           sudo apt install curl jq
           npm install postcss-cli autoprefixer
       - name: 🤵 Install Hugo
         run: |
-          HUGO_VERSION=$(curl -s https://api.github.com/repos/gohugoio/hugo/releases/latest | jq -r '.tag_name')
+          HUGO_VERSION='v0.109.0'
```
Comment (victoriadrake) [inline hugo-latest-cd.yml]: Same question — pinned version?

**Review by victoriadrake — COMMENTED** [review]
Thank you for this! I had a couple questions.

File: action.sh (hunk 1 above)
Comment (IngridMorstrad) [reply action.sh]: My bad, didn't realize all future commits to the branch would get added to the pull request. Reverted, thanks! :)

File: hugo-latest-cd.yml (hunk 2 above)
Comment (IngridMorstrad) [reply hugo-latest-cd.yml]: Same as other comment/reply. Thanks!

**Review by victoriadrake — APPROVED** [review]

**Pull request merged**